src/robot/parsing/model.py

```diff
@@ -597,20 +597,33 @@ def __iter__(self):
 
 
 class ForLoop(_WithSteps):
+    """
+    py:class:: ForLoop(declaration, comment=None)
+
+    :param list declaration: The literal cell values that declare the loop
+                             (excluding ":FOR").
+    :param str comment: A comment, default None.
+    :ivar str flavor: The value of the 'IN' item, uppercased.
+                      Typically 'IN', 'IN RANGE', 'IN ZIP', or 'IN EUMERATE'.
+    :ivar list vars: Variables set per-iteration by this loop.
+    :ivar list items: Loop values that come after the 'IN' item.
+    :ivar str comment: A comment, or None.
+    :ivar list steps: A list of steps in the loop.
+    """
```

Comment:
I think I already commented somewhere else that you can generate API docs by running `doc/api/generate.py`. I checkout out this PR and was able to run it just fine with these changes and the result looks good too. Notice the two comments above, though.